copanier/models.py

```diff
@@ -509,7 +509,9 @@ def incoming(cls):
 
     @classmethod
     def former(cls):
-        return [d for d in cls.all() if not d.is_foreseen]
```

Comment:
I find it hard to read to have the `sorted` and list comprehension on the same line. It's not awful, but I would prefer to have this on two lines, with an intermediate variable. Something like this:

```python
former_deliveries = [d for d in cls.all() if not d.is_foreseen]
return ordered(former_deliveries, key=lambda d: d.from_date, reverse=True)
```

What do you think?

Reply:
Hey yeah, makes tons of sense. I stole the syntax from the method above because my Python days are very very far behind but I refactored both methods to avoid the nesting. Thanks!